contracts/contracts/FleekERC721.sol

```diff
@@ -180,13 +197,13 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
     ) internal virtual override {
         if (from != address(0) && to != address(0)) {
             // Transfer
-            _clearAllTokenRoles(tokenId, to);
+            _clearTokenRoles(tokenId);
         } else if (from == address(0)) {
             // Mint
-            _grantTokenRole(tokenId, Roles.Owner, to);
+            // TODO: set contract owner as controller
```

Comment:
Why is this needed?

Reply:
This needs to be set if we decided that Fleek or the contract owner should be controller for tokens, allowing to verify it. This is one thing we have not setup yet, which address should be bound as the verifier or controller that verifies each token.